src/log-store/src/kafka.rs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2023 Greptime Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod log_store;
+
+use store_api::logstore::entry::{Entry, Id as EntryId};
+use store_api::logstore::namespace::Namespace;
+
+use crate::error::Error;
+
+type Topic = String;
+
+#[derive(Debug, PartialEq, Eq, Hash, Clone)]
```

Comment:
I don't want to repeat myself. I think all the pub structures must have a doc to describe them.

Reply:
Added.